core/src/local_cluster.rs

```diff
@@ -465,6 +466,7 @@ mod test {
         solana_logger::setup();
         let mut fullnode_config = FullnodeConfig::default();
         fullnode_config.rpc_config.enable_fullnode_exit = true;
+        fullnode_config.storage_rotate_count = STORAGE_ROTATE_TEST_COUNT;
```

Comment:
This test fails because 16 ticks per slot seems like its too many and the replicator times out at startup.

Reply:
@sakridge, this is really only an issue during replicator startup alongside the cluster. I don't see a clear path forward in terms of fixing it. Any high ticks_per_slot will delay the generation of the first root slot and cause replicators to time out. 

Right now I'm thinking of bumping the replicator startup delay to cover this case and add a note to the cluster config to say that high values for ticks per slot will cause replicators to timeout. 

Is that acceptable ?

Reply:
We could just change it to 8? or lower?

Reply:
May need to match that with a change to the STORAGE_ROTATE_TEST_COUNT

Reply:
Yeah that would fix the test but also hides that fact that replicators will fail if someone runs with a larger number. Perhaps a note on ticks_per_slot is enough?

Reply:
Yea I think that's fine. I guess ideally the replicator could ask the network for ticks_per_slot and adjust it's timeout accordingly, but that's probably not a high priority change. Most people not running replicator-specific tests will probably not be messing with ticks_per_slot.